src/components/HeaderForm.js

```diff
@@ -294,6 +276,7 @@ class HeaderForm extends Component {
             bedSelect: bedSelect,
             dataPath: dataPath,
             region: ds.region,
+            regionInfo: ds.bedFile ? this.state.regionInfo : {},
```

Comment:
Why should this work? It looks like this is going to keep around the currently-loaded BED info if the data source we are changing to has *any* BED file set, even if it isn't the current one.

This line probably needs an explanatory comment to justify why it needs to be the way it is.

Reply:
The intention was that if a bedFile exists, we can keep the regionInfo was just set by getBedRegions but I'll put another call to setState if it's empty to be more clear

Reply:
Oh OK, there are multiple `setState` calls going on here. That makes sense.